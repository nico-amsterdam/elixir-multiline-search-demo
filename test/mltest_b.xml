<?xml version="1.0" encoding="utf-8"?>
<root>
	<item name"item b">
		<definition />
		<Templates>
			<tpl name="tpl4">tpl text4</tpl>
			<!-- here comes a nested template -->
			<Templates>
				<tpl name="tpl5">tpl text5</tpl>
			</Templates>
			<tpl name="tpl6Â¨>this one will be missed out</tpl>
		</Templates>
		<properties />
	</item>
</root>
