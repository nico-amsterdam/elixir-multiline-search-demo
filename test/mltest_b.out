<Templates>
			<tpl name="tpl4">tpl text4</tpl>
			<!-- here comes a nested template -->
			<Templates>
				<tpl name="tpl5">tpl text5</tpl>
			</Templates>
